src/components/ActionMenu/components/Actions/Actions.tsx

```diff
@@ -73,17 +73,19 @@ export function Actions({actions = [], groups = []}: Props) {
     }
 
     let currentAvailableWidth = availableWidthRef.current;
-
     let newShowableActions: MenuActionDescriptor[] = [];
     let newRolledUpActions: (MenuActionDescriptor | MenuGroupDescriptor)[] = [];
+    const actionsAndGroups = [...actions, ...groups];
 
-    [...actions, ...groups].forEach((action, index) => {
+    actionsAndGroups.forEach((action, index) => {
+      const isSingleAction = actionsAndGroups.length === 1;
```

Comment:
Do we even need to do this loop at all if there is only one action? Could we instead just set it as the showable action in that case?